eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/configuration/ImportDbsnpVariantsJobConfigurationFailValidationStepTest.java

```diff
@@ -30,11 +31,18 @@
 import uk.ac.ebi.eva.accession.dbsnp.test.BatchTestConfiguration;
 import uk.ac.ebi.eva.accession.dbsnp.test.TestConfiguration;
 
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
 import static org.junit.Assert.assertEquals;
+import static uk.ac.ebi.eva.accession.dbsnp.configuration.BeanNames.IMPORT_DBSNP_VARIANTS_STEP;
+import static uk.ac.ebi.eva.accession.dbsnp.configuration.BeanNames.VALIDATE_CONTIGS_STEP;
 
 @RunWith(SpringRunner.class)
 @ContextConfiguration(classes = {BatchTestConfiguration.class, TestConfiguration.class})
-@TestPropertySource("classpath:validate-contigs.properties")
+@TestPropertySource("classpath:application.properties")
```

Comment:
It is a bit counter-intuitive that the step tests now use `application.properties`, and the whole job uses `validate-contigs.properties`

Reply:
What do you mean by "step tests"? Each of the test classes for the job (`ImportDbsnpVariantsJobConfigurationFailValidationStepTest` and `ImportDbsnpVariantsJobConfigurationTest`) is using a different properties file to test the case where the validation fails and when the validation is successful.

I am not sure if you got confused for the word `Step` in `ImportDbsnpVariantsJobConfigurationFailValidationStepTest`?

Reply:
Yes, I got a bit confused by that. But still, given that `ImportDbsnpVariantsJobConfigurationFailValidationStepTest` refers to the validation and the other to the whole job, I would find more intuitive to name the file `validate-contigs.properties` for the former.